Tailviewer.AcceptanceTests/BusinessLogic/LogFiles/TextLogFileAcceptanceTest.cs

```diff
@@ -355,7 +355,7 @@ public void TestRead2LogEntries()
 						new LogFileSection(5, 1)
 					});
 
-				file.GetValue(LogFileProperties.StartTimestamp).Should().Be(new DateTime(2015, 10, 7, 19, 50, 58, 982));
```

Comment:
This merge doesn't look right, GetValue(...) is the new way to retrieve data, please don't revert that.